server/bleep/src/lib.rs

```diff
@@ -281,14 +282,17 @@ impl Application {
                 Err(e) => bail!(e),
             };
 
-        let segment = if let Ok(segment_key) = std::env::var("SEGMENT_KEY_BE") {
-            info!("initializing segment");
-            Some(Segment::new(segment_key))
+        let analytics_client = if let (Ok(key), Ok(data_plane)) = (
+            std::env::var("ANALYTICS_KEY_BE"),
```

Comment:
should we update github workflow ?

Reply:
these are injected at runtime rather than build time, similar to how `segment` operated. i wonder if this is better off injected during build time? cc @ggordonhall @rsdy.

Reply:
the tauri-release flow needs to be updated. https://github.com/BloopAI/roadmap/issues/377